src/backend/joanie/core/models/accounts.py

```diff
@@ -22,6 +22,16 @@ class Meta:
     def __str__(self):
         return self.username
 
+    @staticmethod
+    def update_or_create_from_request_user(request_user):
+        """Create user from token or update it"""
+        user = User.objects.update_or_create(
+            username=request_user.username,
+            defaults={"email": request_user.email},
```

Comment:
I think we should take the opportunity to also store the full name.

Reply:
do we have already the full name from the token or does it needs to be added from Fonzie?
It can be added in other PRs as well.
I have to add the language as well.

Reply:
We already have full_name within the JWT Token but you are right, we could treat this topic in a separated PR as we have